src/index.html

```diff
@@ -7,6 +7,43 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--calendar-length--28">
```

Comment:
```suggestion
    <div class="calendar calendar--sun calendar--28">
```
Можно вот так оставить